src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 20px;
+  background-color: black;
+}
+
+.border {
+  border: 1px solid #e75551;
+}
+
+.bottom,
+.middle,
+.top {
+  box-sizing: border-box;
+  padding: 10px;
```

Comment:
you can add `border` here too instead of to create a class

Reply:
pls explain me why? if i add border to these classes,  we have only 2 border line.